openml/flows/flow.py

```diff
@@ -332,32 +341,38 @@ def publish(self):
         self : OpenMLFlow
 
         """
+        import openml.flows.functions
```

Comment:
This is not possible because of cyclic dependencies. In particular, `flow.py` tries to import `functions.py` in order to call `get_flow()`, while `functions.py` tries to import `flow.py` in order to instantiate an `OpenMLFlow`. I will add a comment.